debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/QueueSegment.java

```diff
@@ -126,6 +125,7 @@ private HashMap<String, Object> generateCdcMessageForRecord(Record r) {
         cdcInfo.put("key", key);
         cdcInfo.put("fields", fields);
         cdcInfo.put("exporter_role", exporterRole);
+        cdcInfo.put("cacheMetadata", r.cacheMetadata);
```

Comment:
For consistency, separate the key using underscore i.e. `"event_id"`

Reply:
Done